system-tests/test/issue_5016_spec.js

```diff
@@ -0,0 +1,13 @@
+const systemTests = require('../lib/system-tests').default
+
+describe('e2e issue 5016 - screenshot times out after clicking target _blank', function () {
+  systemTests.setup()
+
+  systemTests.it('fails but does not timeout taking screenshot', {
+    project: 'config-screenshot-on-failure-enabled',
+    sanitizeScreenshotDimensions: true,
+    snapshot: true,
+    expectedExitCode: 1,
+    browser: '!webkit',
```

Comment:
I only run into the issue in `chrome`. `firefox` and `electron` both seem to work as expected prior to the fix. 
```suggestion
    browser: 'chrome',
```

Reply:
could also be good to just exercise the behavior since it is a change to CDP automation itself and make sure the behavior doesn't regress in those browsers for whatever reason